code/frameworks/svelte-vite/package.json

```diff
@@ -55,7 +55,7 @@
     "@storybook/builder-vite": "7.0.0-beta.3",
     "@storybook/node-logger": "7.0.0-beta.3",
     "@storybook/svelte": "7.0.0-beta.3",
-    "@sveltejs/vite-plugin-svelte": "^1.0.0",
```

Comment:
This upgrade is unrelated to the fix, but a good idea now that we want to support Vite 4